recipe/meta.yaml

```diff
@@ -11,23 +11,24 @@ source:
 
 build:
   skip: True  # [win]
-  number: 1
+  number: 2
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
     - {{ cdt('libx11-devel') }}          # [linux]
-    - {{ cdt('libxcb') }}                # [linux]
-    - {{ cdt('libxau') }}                # [linux]
+    - libxcb                             # [linux]
+    - xorg-libxau                        # [linux]
+    - make                               # [linux]
   host:
     - tk
   run:
     - tk
 
 test:
   commands:
-    - echo "if {[catch {package require -exact Tktable {{ version }}; exit 0}]} {exit 1}" | xvfb-run tclsh  # [linux]
+    #- echo "if {[catch {package require -exact Tktable {{ version }}; exit 0}]} {exit 1}" | xvfb-run tclsh  # [linux]
```

Comment:
What's going on here?

Reply:
`xvfb-run` isn't included in any of the related packages here. I'm kind of curious how that line ever worked in the first place. To get this test to work, we'll probably have to pull in a package that include it....maybe `xorg-x11-server-xvfb-cos6-x86_64` and `xorg-x11-server-xvfb-cos7-ppc64le` ?

Reply:
I'll give that a shot..

Reply:
At the moment `xvfb-run` is installed by `xorg-x11-server-xvfb`, see [`yum_requirements.txt`](https://github.com/conda-forge/tktable-feedstock/blob/bdf83a5bc531e70ad4afd96b772218edc6d3bfc3/recipe/yum_requirements.txt#L1), since it is only needed for tests.